changelog/unreleased/bugfix-shares-tree-loading

```diff
@@ -0,0 +1,12 @@
+Bugfix: Shares tree loading
+
+We've improved loading of the shares tree:
+
+* It now happens more globally in the sidebar component instead of in each sidebar panel.
+* Shares won't be loaded for resources without a path anymore.
+
+These changes massively improve the sidebar performance and fix several issues with (re-)share permissions.
+
+https://github.com/owncloud/web/issues/7506
+https://github.com/owncloud/web/issues/7593
+https://github.com/owncloud/web/pull/7580
```

Comment:
I added https://github.com/owncloud/web/issues/7592 to the PR description. Could you add it to the changelog as well?